searchapp/models.py

```diff
@@ -21,9 +21,37 @@ class Mentor(models.Model):
     def __str__(self):
         return self.username
 
+class Board(models.Model):
+    BOARD_TYPE_CHOICES = (
+        ('CBSE',  'CBSE'),
+        ('ICSE', 'ICSE'),
+    )
+    board = models.CharField(max_length=4,choices=BOARD_TYPE_CHOICES, default='CBSE',unique=True)
+
+    def __str__(self):
+        return self.board
+
+class MinMaxFloat(models.IntegerField):
+    def __init__(self, min_value=None, max_value=None, *args, **kwargs):
+        self.min_value, self.max_value = min_value, max_value
+        super(MinMaxFloat, self).__init__(*args, **kwargs)
+
+    def formfield(self, **kwargs):
+        defaults = {'min_value': self.min_value, 'max_value' : self.max_value}
+        defaults.update(kwargs)
+        return super(MinMaxFloat, self).formfield(**defaults)
+
+class Grade(models.Model):
+    grade = MinMaxFloat(min_value=1, max_value=12, default=10)
```

Comment:
Is this really needed? Can't we have server validation for this instead.

Reply:
This just helps in keeping the grades within this range even if someone tries to add new grades to the database from django admin panel.